packages/tooltips/src/elements/Tooltip.spec.tsx

```diff
@@ -35,47 +36,59 @@ describe('Tooltip', () => {
   });
 
   it('renders tooltip with RTL styling if provided', () => {
-    const { getByTestId } = renderRtl(<BasicExample />);
+    const { getByTestId, getByText } = renderRtl(<BasicExample />);
+
+    expect(getByText('Trigger')).not.toHaveFocus();
 
     act(() => {
-      fireEvent.focus(getByTestId('trigger'));
+      userEvent.tab();
       jest.runOnlyPendingTimers();
     });
 
+    expect(getByText('Trigger')).toHaveFocus();
     expect(getByTestId('tooltip')).toHaveStyleRule('direction', 'rtl');
   });
 
   describe('Focusable Tooltip Content', () => {
     it('remains visible if tooltip content is focused', () => {
```

Comment:
The original test did not have focusable content. The updated test more accurately asserts when a user focuses into the tooltip, and then focuses on a node that is part of the content.